core/templates/dev/head/pages/exploration_player/take_break_modal.html

```diff
@@ -0,0 +1,17 @@
+<script type="text/ng-template" id="modals/takeBreak">
+  <div class="modal-header">
+    <h3>
+      Try taking a break.
```

Comment:
I played it. The text feels a bit directive... perhaps something like the following?

---
**Header**:
Time for a break?

**Body**:
You seem to be submitting answers very quickly. Are you starting to get tired?

If so, consider taking a break! You can come back later.

---

Maybe one day we can supplement this with a fun graphic, too :)